src/components/Sidebar/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import Box from '../Box';
+
+import './style.scss';
+
+const Sidebar = () => {
+  const currentUser = JSON.parse(localStorage.getItem('currentUser'));
+  const userImage = 'https://www.laprensa.com.ar/multimedios/imgs/123027_620.jpg';
+  if (!currentUser) {
+    return null;
+  }
+  return (
+    <div className="sidebar">
+      <Box>
+        <div className="sidebar-header">
+          <img src={userImage} alt="user_profile_picture" />
```

Comment:
```suggestion
          <img src={userImage} alt="User Profile" />
```

Reply:
The alt text serves as a description of the image's content for those who may not be able to see the image, such as people using screen readers.  Check here for more information: https://blog.neocoast.com/web-accessibility

Reply:
Thanks for the data but I prefer the information of Lia